### ISSUE TITLE
Using renovate to manage sonar version

### ISSUE THREAD

**Issue by nimmoadam** [renovatebot/config-help#794]
Which Renovate are you using?

**Renovate Open Source CLI**

Which platform are you using?

**GitHub Enterprise**

Is there a way to use renovate bot to manage sonar version.? If yes, how do we do it? 

Also, I am aware that renovate bot supports only specific languages, but is there a way to use renovate bot to manage some packages, that are used inside a shell script? 

Any help is appreciated. thanks@

**Comment by rarkins** [comment]
Can you elaborate on what you mean? I'm not familiar with sonar version

**Comment by viceice** [comment]
You can use regex manager to update arbitrary text files

**Comment by nimmoadam** [comment]
@rarkins Sonar version is, sonarqube I meant. It is used for inspecting code quality. 
So, we are having separate shell script for running the sonar scan. In the shell script we have mentioned the sonarqube scanner's version number. So, if we want to update that using renovate bot - how can we do? 


@viceice - any pointers or examples please?

**Comment by rarkins** [comment]
There needs to be a datasource which can retrieve all the versions. Eg GitHub tags, maven central, etc. does that exist?

**Comment by nimmoadam** [comment]
@rarkins Nope. It is just a shell script, where the version is mentioned and when the script gets executed this version will be used for running the sonar scan.

**Comment by rarkins** [comment]
You misunderstood my question. Where can you get a list of all available versions from?

**Comment by nimmoadam** [comment]
Hi @rarkins Sorry forgot to share details. 

This is the place where we get different versions of sonarqube - https://binaries.sonarsource.com/Distribution/sonar-scanner-cli/
